metacat-connector-polaris/src/main/java/com/netflix/metacat/connector/polaris/store/PolarisStoreService.java

```diff
@@ -0,0 +1,68 @@
+package com.netflix.metacat.connector.polaris.store;
+
+import com.netflix.metacat.connector.polaris.store.entities.PolarisDatabaseEntity;
+import com.netflix.metacat.connector.polaris.store.entities.PolarisTableEntity;
+import java.util.List;
+import java.util.Optional;
+
+/**
+ * Interface methods for Polaris Store CRUD access.
+ */
+public interface PolarisStoreService {
```

Comment:
Could a delete method for database also be added?

Reply:
sure! will add in next pr. will get it merged and iterate.